ui/apps/dashboard/src/components/Events/EventInfo.tsx

```diff
@@ -0,0 +1,26 @@
+import { Link } from '@inngest/components/Link/Link';
```

Comment:
All these events component here and below are basically the same, I just moved them out of the app router to here to clean things up a abit.